cpp/src/aztec/plonk/proof_system/widgets/random_widgets/random_widget.hpp

```diff
@@ -1,5 +1,5 @@
 #pragma once
-#include "../../../transcript/transcript_wrappers.hpp"
+#include "../../../../transcript/transcript.hpp"
```

Comment:
This file only needed "transcript.hpp" which it was previously getting through "transcript_wrappers.hpp"

Reply:
I've made this change in 1-2 other files and also deleted several includes of transcript_wrappers.hpp wherever possible

Reply:
OK